ts/live-compositor/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import View, { ViewProps } from './components/View';
+import Image, { ImageProps } from './components/Image';
+import Text, { TextProps } from './components/Text';
+import InputStream, { InputStreamProps } from './components/InputStream';
+import Rescaler, { RescalerProps } from './components/Rescaler';
+import WebView, { WebViewProps } from './components/WebView';
+import Shader, { ShaderParam, ShaderParamStructField, ShaderProps } from './components/Shader';
+import Tiles, { TilesProps } from './components/Tiles';
+import { EasingFunction, Transition } from './components/common';
+
+export { RegisterInput } from './types/registerInput';
+export { RegisterOutput } from './types/registerOutput';
+
+export * as Inputs from './types/registerInput';
+export * as Outputs from './types/registerOutput';
+export * as Api from './api';
+
+export { SceneBuilder, SceneComponent } from './component';
+
+export {
+  View,
+  ViewProps,
+  Image,
+  ImageProps,
+  Text,
+  TextProps,
+  InputStream,
+  InputStreamProps,
+  Rescaler,
+  RescalerProps,
+  WebView,
+  WebViewProps,
+  Shader,
+  ShaderProps,
+  Tiles,
+  TilesProps,
+};
+
+export { ShaderParam, ShaderParamStructField, EasingFunction, Transition };
```

Comment:
I'm not sure about that, problem here is that types are partially defined in API and partially overiden here